.gitignore

```diff
@@ -1,2 +1,3 @@
 node_modules
-.idea
\ No newline at end of file
+.idea
+package-lock.json
```

Comment:
You shouldn't ignore the `package-lock.json`, as it helps reproduce the state of this npm library.